records_mover/db/redshift/unloader.py

```diff
@@ -26,11 +27,20 @@ class RedshiftUnloader(Unloader):
     def __init__(self,
                  db: Union[sqlalchemy.engine.Engine, sqlalchemy.engine.Connection],
                  table: Callable[[str, str], Table],
-                 temporary_s3_directory_loc: Callable[[], ContextManager[BaseDirectoryUrl]],
+                 s3_temp_base_loc: Optional[BaseDirectoryUrl],
                  **kwargs) -> None:
         super().__init__(db=db)
         self.table = table
-        self.temporary_s3_directory_loc = temporary_s3_directory_loc
+        self.s3_temp_base_loc = s3_temp_base_loc
+
+    # TODO: Do I want to combine with one below?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker